anesthetic/plot.py

```diff
@@ -43,6 +43,8 @@ class AxesSeries(Series):
     labels : dict(str:str), optional
         Dictionary mapping params to plot labels.
         Default: params
+    logx : list(str), optional
+        List of parameters to be plotted on a log scale.
```

Comment:
Is this the best API? Ideally we'd want to allow for (eventual, probably not this PR) functionality for [symlog or logit](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.xscale.html) scales as well.

How about `xscale` as a keyword argument, which can function as a defaultdict between parameters and keywords (with the default being `'linear'`.

Reply:
`logx` is a [standard pandas kwarg](https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.DataFrame.plot.html#pandas-dataframe-plot), so that's why I picked that... 

I did indeed start out with an `xscale` kwarg, but then stumbled over pandas handling of this and switched... What do we prefer?

Reply:
> `logx` is a [standard pandas kwarg](https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.DataFrame.plot.html#pandas-dataframe-plot), so that's why I picked that...

I had not clocked this, which makes this much more convincing. Stick with logx until pandas also supports symlog/logit.

We can always add an xscale kwarg if necessary to override, but for now we should go with  the general principle of closer pandas mirroring (#59).